department-of-veterans-affairs/caseflow
# Fix bug preventing selection of unlisted claimant radio button

### Description
This PR fixes a frontend bug in how the Intake Review page handles a new unlisted claimant from the Add Claimant modal.

Previously, the new radio button would not get selected, because of an undefined/null mismatch on the claimant value (participant id). In any case, RadioField complains when its values aren't strings, so I've made sure that the RadioField always gets an empty string instead.

### Testing Plan
1. While performing a Board intake, add an unlisted claimant via the Add Claimant modal.
2. Confirm that new radio button for the unlisted claimant is selected.
3. Confirm that it can be re-selected after selecting a different dependent.

### User Facing Changes
Screenshot of a selectable unlisted claimant on the Review page:
<img width="559" src="https://user-images.githubusercontent.com/282869/89454856-5227db80-d72f-11ea-8938-8b944ad1e1de.png">
